src/sql/workbench/contrib/notebook/browser/outputs/gridOutput.component.ts

```diff
@@ -119,6 +119,29 @@ export class GridOutputComponent extends AngularDisposable implements IMimeCompo
 		}
 		if (!this._table) {
 			let source = <IDataResource><any>this._bundleOptions.data[this.mimeType];
+			let columnNames: Array<string> = [];
```

Comment:
Map can be used here
```			
let columnNames: string[] = source.schema.fields.map((field) => {
	return field.name;
});
```

Reply:
or even shorter 
` const columnNames: string[] = source.schema.fields.map(field => field.name);`